components/ViewHabits.tsx

```diff
@@ -12,38 +14,78 @@ const styles = {
     height: '100vh',
     maxHeight: '100vh',
     backgroundColor: '#111',
+    margin: '0',
+  }),
+  header: css({
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    margin: '0 auto',
+    height: '130px',
+    maxWidth: '500px',
+    position: 'relative',
   }),
   logoContainer: css({
-    height: '20vh',
+    height: '200px',
+    width: '100vw',
+    position: 'relative',
   }),
   logo: css({
     maxHeight: '16vh',
+    maxWidth: '100vw',
+  }),
+  habitsContainer: css({
+    maxWidth: '500px',
+    margin: '0 auto',
+  }),
+  addHabitButton: css({
+    borderRadius: '7px',
+    position: 'absolute',
+    padding: '0',
+    height: '48px',
+    width: '48px',
+    top: '65px',
+    right: '8px',
+    transform: 'translateY(-50%)',
   }),
 };
 
 const ViewHabits = () => {
   const [habitsData, setHabitsData] = useState<GoalWithHabitHistory[]>([]);
 
   useEffect(() => {
-    fetchData().then(setHabitsData);
+    habitsApi.getHabits().then(setHabitsData);
```

Comment:
[non-blocking] [curious] how should we handle errors?

Reply:
I'd ideally like to make an error component or use a library for it so you can just send the error to that system and it can display messages to the user like a toast with actions?

Reply:
Great idea. There are definitely options for this - file a ticket and investigate ;)